grails-app/services/org/pih/warehouse/core/DocumentService.groovy

```diff
@@ -1635,4 +1635,35 @@ class DocumentService {
         return documents
     }
 
+    void generateAmountOutstandingOnOrdersxcel(OutputStream outputStream, List<Map> data) {
+        try {
+            Workbook workbook = new HSSFWorkbook()
+            HSSFSheet sheet = workbook.createSheet("Sheet1")
+            createExcelHeader(sheet, 0, data.get(0).keySet().toList())
+            data.eachWithIndex { Map dataRow, index ->
+                createExcelRow(sheet, index + 1, dataRow)
+            }
+
+            sheet.setColumnWidth((short) 0, (short) ((50 * 4) / ((double) 1 / 20))) // Supplier
```

Comment:
@jmiranda what is behind this width calculation? I used this as it is common within this service but I am curious about what it is based on.

Reply:
I don't remember, but there was some reason for it or else I wouldn't have done it. I'd love to remove it though.

Reply:
It would be great to either remove it or make it generic so we don't need more than one generateExcel() methods.

Reply:
I don't need this, I prefer the generic version. This was requested by QA. I'll ask Kelsey on the standup if this is something she wants.